.devcontainer.json

```diff
@@ -1,7 +1,7 @@
-// See https://aka.ms/vscode-remote/devcontainer.json for format details.
-{
-    "dockerFile": "Dockerfile",
-    "runArgs": [
-        "-v", "C:\\Users\\phili\\Documents\\GitHub\\optee_website:/srv"
-    ]
-}
+// See https://aka.ms/vscode-remote/devcontainer.json for format details.
+{
+    "dockerFile": "Dockerfile",
+    "runArgs": [
+        "-v", "C:\\Users\\phili\\Documents\\GitHub\\optee_website:/srv"
```

Comment:
runArgs actually needs to come out. I only needed it for testing and I shouldn't have committed it in the first place.